test/game-test.js

```diff
@@ -49,9 +50,15 @@ describe('GAME CLASS', function() {
     expect(game.roundCount).to.equal(2);
   });
   
-  it('Should be able to instantiate a new round', () => {
+  it.only('Should be able to instantiate a new round', () => {
+    game.makePlayers('Kirk', 'Ayla')
+    game.getSurveys();
     game.startRound();
-    expect(game.round).to.be.an.instanceof(Round);
+    console.log('ROUND! ',Round)
+
+    // expect(game.round).to.be.an.instanceOf(Round);
+    console.log("hi: ", game.round instanceof Round)
```

Comment:
If you just instantiate a new round outside of the game context, would the test pass? There could be something strange going on with that assertion method being used in the context of the game?

I would also check what `game.round.prototype` or `game.round.__proto__` gives you back